velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -1181,3 +1181,37 @@ TEST_F(ParquetReaderTest, testV2PageWithZeroMaxDefRep) {
   assertReadWithReaderAndExpected(
       outputRowType, *rowReader, expected, *leafPool_);
 }
+
+TEST_F(ParquetReaderTest, testLzoDataPage) {
+  const std::string sample(getExampleFilePath("lzo.parquet"));
+
+  facebook::velox::dwio::common::ReaderOptions readerOptions{leafPool_.get()};
```

Comment:
nit: we should add tests to a "facebook::velox::dwio::test" sort of namespace, so you could just do "common::ReaderOptions" here